src/main/java/edu/cscie71/imm/app/slacker/client/SlackerClient.java

```diff
@@ -1,7 +1,22 @@
 package edu.cscie71.imm.app.slacker.client;
 
+import org.apache.http.client.ClientProtocolException;
+import org.apache.http.client.fluent.Request;
+import org.apache.http.NameValuePair;
+import java.util.List;
+import java.io.IOException;
+
 public class SlackerClient implements ISlackerClient {
-    public boolean postMessage(String token, MessagePost message) {
-        return true;
+
+    public String postMessage(String token, MessagePost message) throws IOException, ClientProtocolException {
```

Comment:
Not yet. In retrospect we should catch the exceptions and, at the very least, return a json string indicating the thrown exception.